src/interactions/serialization/txOutput.ts

```diff
@@ -105,3 +116,28 @@ export function serializeTxOutputDatum(
     }
 }
 
+export function serializeTxOutputScriptHex(
```

Comment:
```suggestion
export function serializeTxOutputRefScript(
```
Definitely no "hex" in the name of this function.